### ISSUE TITLE
Motion Effects generates js error

### ISSUE THREAD

**Issue by adverman** [elementor/elementor#12299]
In Elementor 3.0.x adding motion effect to a column generates a js error that breaks others scripts.

![image](https://user-images.githubusercontent.com/8201505/91549543-63779880-e927-11ea-93a0-3d36c07bb902.png)

In consolle:
```
Uncaught TypeError: Failed to execute 'observe' on 'IntersectionObserver': parameter 1 is not of type 'Element'.
    at n.value (frontend.min.js?ver=3.0.1:2)
    at n.value (frontend.min.js?ver=3.0.1:2)
    at init (frontend-modules.min.js?ver=3.0.3:2)
    at n.Module (frontend-modules.min.js?ver=3.0.3:2)
    at new child (frontend-modules.min.js?ver=3.0.3:2)
    at n.<anonymous> (frontend.min.js?ver=3.0.1:2)
    at new n (frontend.min.js?ver=3.0.1:2)
    at n.<anonymous> (frontend.min.js?ver=3.0.1:2)
    at new n (frontend.min.js?ver=3.0.1:2)
    at Object.<anonymous> (frontend.min.js?ver=3.0.1:2)
```

**Comment by PuxianAlHazred** [comment]
+1

**Comment by adverman** [comment]
#12275 Duplicated
https://github.com/elementor/elementor/issues/12275

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v3.0.2

Feel free to update

Thanks!
